### ISSUE TITLE
Now only converts in one direction

### ISSUE THREAD

**Issue by symmetriq** [js2coffee/js2coffee#346]
The old web interface used to allow conversion from either language to the other. I used to use js2coffee.org to check the JS output by pasting CoffeeScript into the right pane. It seems in 2.0 the conversion now only works from JS to CoffeeScript (hence the name, yes, but for me this is a loss of a very useful feature that I will now have to find another solution for). For some reason the right pane is still "editable", but there seems to be no point in typing anything into that side.

Seems weird to intentionally cripple it by removing such a useful feature.

**Comment by rstacruz** [comment]
It still works... just click on the CoffeeScript pane and keep typing.

**Comment by rstacruz** [comment]
> but there seems to be no point in typing anything into that side.

can I ask why there seems to be no point in it? not taking a jibe or anything—just plain curious on what makes you feel that way.

**Comment by coostendorp** [comment]
I keep a js2coffee tab open all the time, and paste wherever necessary (sometimes I paste in js, sometimes in coffee). Now, after converting from coffee, I need to remember to close the js overlay all the time. It became harder to use for me :-(

**Comment by rstacruz** [comment]
cmd+1 closes it :)

**Comment by symmetriq** [comment]
The reason I said "no point in typing" is that nothing happens when I type in the right pane. Previously I would just paste in some CoffeeScript and immediately see the JS. Now I paste into the right pane and nothing happens. The left side goes dim, but doesn't display anything. There's an (x), but clicking it just erases everything I pasted into the CS side snd reverts to the original example.

Hence I don't see the purpose in typing anything into the CS pane, since it no longer does anything.

**Comment by symmetriq** [comment]
So either it's not working as intended (at least for me), or the CS to JS functionality was removed. I wasn't sure which.

**Comment by rstacruz** [comment]
now that's a bug. Can you tell me what your set up is (browser, os) and what CS code you're pasting in?

**Comment by symmetriq** [comment]
Hey, I just tried it again and now it's working fine. It looks like what happened is the code I pasted in earlier had a CS syntax error, and I just didn't see the error. Now that I know about the error (I found and fixed it a little while after I opened this thread), if I paste the old version in and look where the error is, I do see a red (x) next to the bad line.

Previously js2coffee would display an obvious error on the JS side if the CS code was bad. Now it seems to display nothing at all. Perhaps the left side should indicate that there's an error on the right so users know to scroll through the right pane and find the bad line. Or, now that I know I might not see an error on the left, I can just scroll the right pane myself to find out what happened.

When I paste in the fixed version, the JS appears immediately, same as before, and the larger text area and syntax highlighting are nice improvements.
